test/realtimecontroller/DefaultRealtimeController.test.ts

```diff
@@ -538,6 +544,44 @@ describe('DefaultRealtimeController', () => {
       expect(callbackFired).to.be.true;
     });
 
+    it('will unsubscribe from a given callback if provided', () => {
+      const rt: RealtimeController = new DefaultRealtimeController();
+      const fooAttendeeId = 'foo-attendee';
+      const sentVolume = 0.5;
+      const sentMuted = false;
+      const sentSignalStrength = 1;
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const state: RealtimeState = ((rt as unknown) as any).state as RealtimeState;
+      state.volumeIndicatorCallbacks = {};
+      const callback1 = (
+        attendeeId: string,
+        volume: number | null,
+        muted: boolean | null,
+        signalStrength: number | null
+      ): void => {
+        expect(attendeeId).to.equal(fooAttendeeId);
+        expect(volume).to.equal(sentVolume);
+        expect(muted).to.equal(sentMuted);
+        expect(signalStrength).to.equal(sentSignalStrength);
+      };
+      const callback2 = (
+        attendeeId: string,
+        volume: number | null,
+        muted: boolean | null,
+        signalStrength: number | null
+      ): void => {
+        expect(attendeeId).to.equal(fooAttendeeId);
+        expect(volume).to.not.equal('');
+        expect(muted).to.not.equal('');
+        expect(signalStrength).to.not.equal('');
+      };
+      rt.realtimeSubscribeToVolumeIndicator(fooAttendeeId, callback1);
+      rt.realtimeSubscribeToVolumeIndicator(fooAttendeeId, callback2);
+      rt.realtimeUnsubscribeFromVolumeIndicator(fooAttendeeId, callback2);
+      expect(state.volumeIndicatorCallbacks[fooAttendeeId].indexOf(callback1)).to.equal(0);
+      expect(state.volumeIndicatorCallbacks[fooAttendeeId].indexOf(callback2)).to.equal(-1);
+    });
```

Comment:
Don't you need a test for attempting to remove an absent callback (e.g., removing twice)?

Reply:
What sort of result would be expected from that test?

Reply:
Mostly I'm surprised you got 100% test coverage without it.

But it should complete with no error, and leave `volumeIndicatorCallbacks` unmodified.

Reply:
Updated. Tried to capture this case.